pages/doc/tas_to_troubleshoot.md

```diff
@@ -127,7 +127,7 @@ You have successfully set up the nozzle and the integration. However, you don't
 
 
 * Ensure that the installation of the Wavefront Nozzle in has completed.
-* Verify that the proxy uses the correct API token and Wavefront instance URL. You specify that information in Ops Manager in the **Proxy Config** page.
+* Verify that the proxy uses the correct Auth Credentials (CSP or legacy API token) and Wavefront instance URL. You specify that information in Ops Manager in the **Proxy Config** page.
```

Comment:
I'd keep it simple here:

Verify that the proxy uses the correct authentication credentials and Operations for Applications instance URL. You specify that information in Ops Manager in the **Proxy Config** page.